src/libs/ResizeSensor/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare class ResizeSensor {
+    constructor(
+        element: Element,
+        resizeListener: (size: { width: number; height: number }) => void,
```

Comment:
I see that resizeListener is used inside attachResizeEvent and there we don't pass any argument, then size argument should be optional